### PR TITLE
Add country code args for Canada

### PR DESCRIPTION
**Pull request by jyeung79**
- Moved skus and country codes to another file
- Australia site does not work

### CODE REVIEW

**Comment by jyeung79** [comment]
I added another args where an user can enter a country code. Currently I added CA and AU. However, AU doesn't work for some reason. I'm sure that the country code `/au/` is correct but it's not returning the expected response.

File: constants.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  'MK1A3LL/A': '16" SG, Best',
+  'MK233LL/A': '16" SG, Ultimate',
+  'MMQW3LL/A': '16" Si, Ultimate',
+  'MYD92LL/A': '13" Control',
+};
+
+const COUNTRIES = {
+  US: "",
+  CA: "/ca",
+  AU: "/au", // Issue with fetching australian stock
```
Comment (worthbak) [inline constants.js:22]: I saw the same issue with fetching Australian data - not sure what's going on there. Browsing around a bit on the `au` Apple Store, my best guess is that some fundamental difference with in-store pickup in Australia vs North America. weird.

**Review by worthbak — CHANGES_REQUESTED** [review]
thanks so much for the changes! great idea to add country-code support, and nice implementation. one request before you merge: could you update the `README.md` noting the additional parameter?

**Comment by jyeung79** [comment]
Okay it has been added into the Readme. Let me know if there's any issues. I'll take a look at whether we can make selecting favorite models easier and simpler

**Review by worthbak — APPROVED** [review]
sick - that's exactly what i was going for. thanks!!

**Pull request merged**